Logged error during failed delete request is always nil

https://github.com/aws/amazon-vpc-cni-k8s/blob/1e2853beaf8473c16413eef81d55f2dedd0f8f4a/plugins/routed-eni/cni.go#L277-L281

This code block logs and wraps an `err` that was proven nil in the block above, so it ends up providing no context. We see this error quite frequently in our cluster (dozens of times per hour per node), and are unable to diagnose the root cause because our error messages appear as:

> [ERROR] Failed to process delete request for pod my-pod namespace my-namespace container 08bbc33771440c1d5a1c1b773d708f31a692484148708102fa94dc2c7686c35c: <nil>`

--- a/plugins/routed-eni/cni.go
+++ b/plugins/routed-eni/cni.go
@@ -276,9 +276,9 @@ func del(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 	}
 
 	if !r.Success {
-		log.Errorf("Failed to process delete request for pod %s namespace %s container %s: %v",
-			string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_INFRA_CONTAINER_ID), err)
-		return errors.Wrap(err, "del cmd: failed to process delete request")
+		log.Errorf("Failed to process delete request for pod %s namespace %s container %s: Success == false",
+			string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_INFRA_CONTAINER_ID))
+		return errors.New("del cmd: failed to process delete request")
 	}
 
 	addr := &net.IPNet{

